src/views/test/operations/filter.ts

```diff
@@ -0,0 +1,175 @@
+import InputFilterInteger from '../components/InputFilterInteger'
+
+import { getNumberFilterOperator, getStringFilterOperator, getDateFilterOperator, getBooleanFilterOperator } from './filterQueries'
+import { GridColTypeDef, getGridNumericColumnOperators, getGridStringOperators, GridFilterOperator } from "@material-ui/data-grid";
+
+let filterData;
+export const getFilterData = (filter) => {
+    filterData = filter
+}
+
+export const filterDataGrid = (onFilterCustomers, onChangePageCustomers) => {
+    const filteredQueryForGraphQl = getQueryFromDataGrid()
+    sendFilterQueryToGraphQl(filteredQueryForGraphQl, onFilterCustomers)
+    setCurrentPageToOne(onChangePageCustomers)
+}
+
+const sendFilterQueryToGraphQl = (filteredQueryForGraphQl, onFilterCustomers) => {
+    const filteredValue: string = filterValueFromDataGrid()
+    if (filteredValue) {
+        setFilteredQueryToGraphQl(filteredQueryForGraphQl, onFilterCustomers)
+    } else {
+        setEmptyFilteredQueryValueToGraphQl(onFilterCustomers)
+    }
+}
+
+const getQueryFromDataGrid = () => {
+    const filteredQueryForGraphQl: object = {}
+    const filterModels = filterModelFromDataGrid()
+
+    filterModels.forEach(filterModel => {
+        console.log(filterData, 'filterDatafilterData');
```

Comment:
There are a lot of console.log, throughout the files, which I will remove before I merge this task